lib/kc_elixir_web/templates/page/index.html.eex

```diff
@@ -15,6 +15,27 @@
       <li>
         <strong>Topic</strong>: <%= meetup.topic %>
       </li>
+      <li>
+        <strong>Speaker Info</strong>:
+        <ul>
+          <li><%= if Map.get(meetup.speaker, :name, nil) do %>
+                Name: <%= meetup.speaker.name %>
+              <% end %>
+          </li>
+          <li><%= if Map.get(meetup.speaker, :url, nil) do %>
+                URL: <a href="<%= meetup.speaker.url %>"><%= meetup.speaker.url %></a>
+              <% end %>
+          </li>
+          <li><%= if Map.get(meetup.speaker, :github, nil) do %>
+                GitHub: <a href="<%= meetup.speaker.github %>"><%= meetup.speaker.github %></a>
+              <% end %>
+          </li>
+          <li><%= if Map.get(meetup.speaker, :twitter, nil) do %>
+                Twitter: <a href="<%= meetup.speaker.twitter %>"><%= meetup.speaker.twitter %></a>
```

Comment:
There's kind of an implication that labels here are wrapped in `<strong>` tags, so we might consider doing so here. (`Twitter`, `GitHub`, etc.)